MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/account/AccountRetrievalService.java

```diff
@@ -61,20 +61,16 @@ protected void onHandleIntent(@Nullable Intent intent) {
   }
 
   private void getAccessToken(String code) {
-
     String clientSecret = getString(R.string.mapbox_auth_flow_secret);
-    Log.d("AccountRetrievalService", "getAccessToken: clientSecret = " + clientSecret);
-
     OkHttpClient client = new OkHttpClient();
-
     Request request = new Request.Builder()
-      .addHeader("User-Agent", "Android Dev Preview")
-      .addHeader("Content-Type", "application/x-www-form-urlencoded")
-      .url(ACCESS_TOKEN_URL)
-      .post(RequestBody.create(MediaType.parse("application/x-www-form-urlencoded"),
-        "grant_type=authorization_code&client_id=" + clientId + "&client_secret=" + clientSecret
-          + "&redirect_uri=" + redirectUri + "&code=" + code))
-      .build();
+        .addHeader("User-Agent", "Android Dev Preview")
+        .addHeader("Content-Type", "application/x-www-form-urlencoded")
+        .url(ACCESS_TOKEN_URL)
+        .post(RequestBody.create(MediaType.parse("application/x-www-form-urlencoded"),
+            "grant_type=authorization_code&client_id=" + clientId + "&client_secret=" + clientSecret
+                + "&redirect_uri=" + redirectUri + "&code=" + code))
```

Comment:
I'm having trouble using `Uri.Builder` to correctly build this URL. I propose moving this to a separate ticket and a future PR...